converters/fromZigbee.js

```diff
@@ -1773,7 +1773,39 @@ const converters = {
             return {action: `${direction}`};
         },
     },
-
+    ZYCT202_on: {
+        cid: 'genOnOff',
+        type: 'cmdOn',
+        convert: (model, msg, publish, options) => {
+            return {action: msg.groupid+'_on'};
```

Comment:
I think it's nicer to have `action: 'on', groupid: msg.groupid` here, perhaps some people don't want to act on the group id, for that case they now have to respond to 5 different actions. I don't see a downside in separating them either.

Reply:
How can I do in Home Assistant in an automation a state check at state 'on' and groupid '45755' then switch 'Y'?
You now assume that people who have this remote control do not use the 1,2,3,4,5 and Group selector?

Reply:
You can do:

```yaml
automation:
  - alias: Respond to button clicks
    trigger:
      platform: mqtt
      topic: 'zigbee2mqtt/<FRIENDLY_NAME'
    condition:
      - condition: template
        value_template: '{{ "on" == trigger.payload_json.action }}'
      - condition: template
        value_template: '{{ 1234 == trigger.payload_json.groupid }}'
    action:
      entity_id: light.bedroom
      service: light.toggle
```

Reply:
Ok. I have to learn to let go the automation rules based on entity and state, and go to work with mqtt topics and conditions.
I changed it...